javalin/src/test/java/io/javalin/TestSse.kt

```diff
@@ -109,14 +108,14 @@ class TestSse {
     fun `sending async data is properly processed`() = TestUtil.test { app, http ->
         app.sse("/sse") {
             it.sendEvent("Sync event")
-            CompletableFuture.runAsync {
+            it.ctx.async {
                 Thread.sleep(100)
                 it.sendEvent("Async event")
-                it.close()
             }
```

Comment:
Does this mean all SSE connections are closed unless there is call to `ctx.async` inside?